src/Arcus.Messaging.Pumps.EventHubs/AzureEventHubsMessagePump.cs

```diff
@@ -142,13 +145,28 @@ private async Task ProcessMessageAsync(ProcessEventArgs args)
 
             // TODO: create extension on `EventData` to retrieve context.
```

Comment:
Is this todo on the backlog ?
If not, we can add it to the backlog and remove this 'todo' ?

Reply:
Yes, added it recently: https://github.com/arcus-azure/arcus.messaging/issues/345
Will remove TODO. 👍

Reply:
Removed in https://github.com/arcus-azure/arcus.messaging/pull/344/commits/4b53213f075c3bda880746c1bd257439c50bbe8d